operator/internal/manifests/openshift/alertingrule.go

```diff
@@ -0,0 +1,34 @@
+package openshift
+
+import lokiv1 "github.com/grafana/loki/operator/apis/loki/v1"
+
+func AlertingRuleTenantLabels(ar *lokiv1.AlertingRule) {
```

Comment:
Why use a pointer here?

Reply:
Because we amend the rule groups inline.